compiler/src/toEC.ml

```diff
@@ -100,8 +100,6 @@ let ec_keyword =
  ; "equiv"
  ; "hoare"
  ; "ehoare"
```

Comment:
Normally, easycrypt should be able to provide this list. I don't know if we should add this dependency.

Reply:
Indeed, we could spend a day do develop a tool that may save 5mn every 6 months…